cmd/admin-info.go

```diff
@@ -187,31 +187,35 @@ func mainAdminInfo(ctx *cli.Context) error {
 	// If the error is not nil and not unrecognizable, just print it and exit
 	fatalIf(probe.NewError(e), "Cannot get service status.")
 
-	// Construct the backend status
-	storageInfo := backendStatus{
-		Total: serverInfo.StorageInfo.Total,
-		Free:  serverInfo.StorageInfo.Free,
-	}
-	if serverInfo.StorageInfo.Backend.Type == madmin.Erasure {
-		storageInfo.Backend = xlBackend{
-			Type:         erasureType,
-			OnlineDisks:  serverInfo.StorageInfo.Backend.OnlineDisks,
-			OfflineDisks: serverInfo.StorageInfo.Backend.OfflineDisks,
+	for _, serverInfo := range serversInfo {
```

Comment:
I addressed your comments @harshavardhana, but getting ridding of humanized-duration.go was not possible right now because humanize.Time() which adds an annoying space due to a bug. We should fork it and try again in a separate PR probably